aten/src/THC/THCNumerics.cuh

```diff
@@ -663,7 +663,7 @@ struct THCNumerics<double> {
   static inline __host__ __device__  double tanh (double a) { return  ::tanh(a); }
   static inline __host__ __device__  double erf  (double a) { return   ::erf(a); }
   static inline __host__ __device__  double erfc (double a) { return  ::erfc(a); }
-  static inline __host__ __device__  double abs  (double a) { return   ::abs(a); }
+  static inline __host__ __device__  double abs  (double a) { return   fabs(a); }
```

Comment:
Probably not the greatest idea to reduce the precision from the double overload of abs to fabs. It's better to add the implementation for the abs double overload directly in the HIP stack in order to maintain parallelism here in the code.

Reply:
It does not reduce precision. http://www.cplusplus.com/reference/cmath/fabs/